src/lru.js

```diff
@@ -84,6 +84,16 @@ class LRU {
 		return result;
 	}
 
+	getExpirationTime (key) {
+		let result;
+
+		if (this.#has(key)) {
```

Comment:
I'm not sure if expiration check should also be added here, so that undefined would be returned for keys that already expired

Reply:
Nah, this exactly what it should do.